### ISSUE TITLE
[Feature Request] Daily Digest option for E-Mail notifications

### ISSUE THREAD

**Issue by schuerg** [jbox-web/redmine_git_hosting#279]
Is it possible to configure this plugin in such a way that it sends all E-Mail notifications only once a day bundled in a single E-Mail and not after every single commit to the git repo?

**Comment by n-rodriguez** [comment]
Hi!
I won't add this feature as it would add too much complexity where there's already a lot.
Sorry for the long response delay.

**Comment by n-rodriguez** [comment]
A Redmine plugin already does this kind of job : https://github.com/Undev/redmine_digest.
Why not plug Redmine Git Hosting into it with some kind of external API to declare events?
I don't know how it's working yet but this sounds like a lead.

**Comment by PowerKiKi** [comment]
As discussed in https://github.com/jbox-web/redmine_git_hosting/issues/732#issuecomment-532362560, issues related to Redmine < 4.0 or severely outdated issues are being closed to help clean up the issue tracker.

If this issue is still relevant to you and you are running Redmine >= 4.0, please open a new issue including all new relevant information.
